src/views/virtualmachines/details/tabs/details/components/Descheduler/Descheduler.tsx

```diff
@@ -1,11 +1,11 @@
 import * as React from 'react';
 
-import { modelToGroupVersionKind } from '@kubevirt-ui/kubevirt-api/console';
-import KubeDeschedulerModel from '@kubevirt-ui/kubevirt-api/console/models/KubeDeschedulerModel';
+// import { modelToGroupVersionKind } from '@kubevirt-ui/kubevirt-api/console';
```

Comment:
comments

Reply:
I want to keep this comments as in a follow up PR I'll take this code and move it to another place